ui/shared/components/panel/variants/SavedVariants.jsx

```diff
@@ -76,14 +78,17 @@ class SavedVariants extends React.PureComponent {
     const {
       match, tableState, filters, discoveryFilters, totalPages, variantsToDisplay, totalVariantsCount, firstRecordIndex,
       tableSummaryComponent, loading, filteredVariantsCount, tagOptions, additionalFilter, updateTableField,
-      variantExportConfig, loadVariants, error,
+      variantExportConfig, loadVariants, error, multiple,
     } = this.props
     const { showAllFilters } = this.state
     const { familyGuid, variantGuid, tag } = match.params
 
-    const appliedTagCategoryFilter = tag || (variantGuid ? null : (tableState.categoryFilter || ALL_FILTER))
+    const tags = tag ? tag.split(TAG_URL_DELIMITER) : tag
+    const tagCategoryFilters = tags || (variantGuid ? [] : [(tableState.categoryFilter || ALL_FILTER)])
+    const firstAppliedTagCategoryFilter = tagCategoryFilters.length > 0 ? tagCategoryFilters[0] : null
+    const appliedTagCategoryFilter = multiple ? tagCategoryFilters : firstAppliedTagCategoryFilter
 
-    let shownFilters = (discoveryFilters && appliedTagCategoryFilter === DISCOVERY_CATEGORY_NAME) ?
+    let shownFilters = (discoveryFilters && firstAppliedTagCategoryFilter === DISCOVERY_CATEGORY_NAME) ?
```

Comment:
similar to above, `discoveryFilters` is page specific so its confusing to have this logic here, it would be better to move this logic into the page-specific component and conditionally pass the correct `filters` prop